distro.py

```diff
@@ -925,16 +925,26 @@ def _lsb_release_info(self):
         Returns:
             A dictionary containing all information items.
         """
-        if not self.include_lsb:
+        cmd = ['lsb_release', '-a']
+        process = subprocess.Popen(
+            cmd,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE)
+        stdout, stderr = process.communicate()
+        stdout, stderr = stdout.decode('utf-8'), stderr.decode('utf-8')
```

Comment:
Let's continue to use `sys.getfilesystemencoding()` instead of  assuming `utf-8` here. Also remove the tuple pack and unpack. Doesn't make the code any more readable IMO.